js/libs/popover.js

```diff
@@ -115,7 +115,21 @@ var $ = require('jquery'),
   }
 
   $(document).ready(function() {
-    $('[data-toggle="popover"]').popover();
+    $('[data-toggle="popover"], [rel="clickover"]')
+    .on('click', function(e) {
+      e.preventDefault();
+      e.stopPropagation();
+    }).popover();
+  });
+
+  $(document).on('click', function (e) {
+    $('[data-toggle="popover"][data-autoclose="true"], [rel="clickover"]').each(function () {
+        //the 'is' for buttons that trigger popups
+        //the 'has' for icons within a button that triggers a popup
+        if (!$(this).is(e.target) && $(this).has(e.target).length === 0 && $('.popover').has(e.target).length === 0) {                
+            (($(this).popover('hide').data('bs.popover')||{}).inState||{}).click = false  // fix for BS 3.3.6
```

Comment:
Should we make this easier to reason about, something like this might work...

```
var $bsPopover = $this.popover('hide').data('bs.popover');

if ($bsPopover !== undefined && $bsPopover.inState) {
    $bsPopover.click = false;
}
```